cookbook/bundles/dependencies.rst

```diff
@@ -0,0 +1,52 @@
+.. index::
+   single: Bundle; Dependencies
+
+How to Use Bundle Dependencies to load other Bundles
```

Comment:
Our titles case standard is to capitialize all words, except from closed-class words. Load is not closed-class (you can say "quickly load" for instance) and has to be capitialized